src/Time/Units.hs

```diff
@@ -135,14 +137,17 @@ type instance UnitName (86400   :% 1) = "d"  -- day unit
 type instance UnitName (604800  :% 1) = "w"  -- week unit
 type instance UnitName (1209600 :% 1) = "fn" -- fortnight unit
 
-instance KnownSymbol (UnitName unit) => Show (Time unit) where
+-- | Constraint alias for 'KnownSymbol' 'UnitName'.
+type KnownUnitSymbol unit = KnownSymbol (UnitName unit)
```

Comment:
Both constraints `KnownUnitSymbol` and `KnownRat` also appear quite often. You can create another constraint like `KnownRatName`. Also, let's rename `KnownUnitSymbol` to `KnownUnitName` to make it shorter :slightly_smiling_face: